### ISSUE TITLE
[low] JAWS+Chrome: Many Pages: Visual headings are not announced as such

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5458]
## Action Performed:
Prerequisites: the user is not logged in.
1. Using JAWS+Chrome, open https://staging.new.expensify.com/
2. Press H key to navigate through the headings.
3. Note what headings the page has.

## Expected Result:
Visually the 'Split bills and chat with friends' text looks like a heading, so it should be also announced as heading level 1.

## Actual Result:
The 'Split bills and chat with friends' text is visually represented as a heading but programmatically it is not implemented as a heading.

HTML:
`<div dir="auto" class="css-901oao" style="color: rgb(255, 255, 255); font-family: GTAmericaExp-Regular; font-size: 36px; font-weight: 700; line-height: 40px;">Split bills and chat with friends.</div>`

Other occurrences:
1. Terms of service -> 'Expensify terms of service', 'Who Can Use the Expensify Service?', 'Individual Account Registration', 'Corporate Policies', 'Linked Financial Accounts', 'Bill Pay and Invoicing', 'Personal Payments', 'Third Party Payment Services', 'Fees and Payment', 'Term and Termination' and many more are visually looking as headings but programmatically they are not defined as headings.
2. Set password -> the visual 'Split bills and chat with friends' heading is not implemented as heading semantically.
3. Email verification -> the visual 'Split bills and chat with friends' heading is not implemented as heading semantically.
4. Create new group overlay -> the visual headings 'New Group', 'Recents' and 'Contacts' are not implemented as headings semantically.
5. Privacy policy -> the visual headings are not implemented as headings semantically.
6. Request money overlay -> the visual 'Request money' heading is not marked up semantically as a heading.
7. Settings overlay -> the visual 'Settings' heading is not implemented as a heading semantically.
8. Split bill overlay -> the visual 'Split Bill' heading is not marked up semantically as a heading.

## Workaround:
No workaround

## Area issue was found in:
Landing page, Terms of service page, Set password page, Email verification page, Create new group overlay, Privacy policy page, Request money overlay, Settings overlay, Split bill overlay

## Failed WCAG checkpoints
1.3.1

## User impact:
The screen reader user might not understand the structure of page.

## Suggested resolution:
Use heading semantics to implement the visual headings, make sure to follow the hierarchy of headings (the main page heading should be implemented as `<h1>`, the section headings as `<h2>`, followed by sub-section headings implemented as `<h3>` and so on). See: https://www.w3.org/WAI/tutorials/page-structure/headings/

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**
![Bug5180527_heading_-_terms_of_service](https://user-images.githubusercontent.com/88733897/134555076-3e60bf04-340f-4a26-925a-519df0721a78.png)

https://user-images.githubusercontent.com/88733897/134555087-9bd5c188-39bc-4a67-a5be-18e1f2e2984e.mp4

![Bug5180527_info_and_relationship_-_split_bills_-_landing_page](https://user-images.githubusercontent.com/88733897/134555092-a57679b3-adfe-4111-9a95-4f29be485fb0.png)
![Bug5180527_Main_heading_-_set_password](https://user-images.githubusercontent.com/88733897/134555105-e7ecfbf8-1f98-47e7-b4fc-1d77578ab17c.png)
![email verification page-visual Heading](https://user-images.githubusercontent.com/88733897/134555118-a1f22cbf-3d6d-4af4-b566-bb705ddc6301.png)
![Jaws_Chrome-Create_New_Group-Visual_headings_are_not_programmatic](https://user-images.githubusercontent.com/88733897/134555124-9e915a55-4296-400a-bf3c-088e424efee3.png)
![Privacy policy page- visual headings](https://user-images.githubusercontent.com/88733897/134555130-9fc03079-9aec-41f5-b8b3-29f3bf721d9e.png)
![Request money page-visual heading](https://user-images.githubusercontent.com/88733897/134555135-3428d990-b1ac-4512-8e5d-617032734872.png)
![Settings page-Visual heading](https://user-images.githubusercontent.com/88733897/134555143-61d19265-4cdf-4336-99e1-5746e6dc5f5b.png)
![Split bills-visual heading](https://user-images.githubusercontent.com/88733897/134555153-c84277e7-466c-49d8-bd58-54dc98ca838c.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
